Automattic/sensei
# Update lesson status preview

### Changes proposed in this Pull Request

- Change lesson preview statuses to "Not Started" and "Completed".
- ~Change UI to ButtonGroup.~ ~_Leaving this as a dropdown based on design feedback. Using a ButtonGroup is not scalable to more options. See p6rkRX-1Ds-p2._~ *Using the shared radio boxes*
- Use the same UI for setting status preview as the modules
- ~Use SVG in CSS for check mark icon~ Use `<svg>` elements in HTML, so it can use text color, same as #3679

### Testing instructions

* Open a course with lessons in the editor
* Toggle the lesson status between 'Not Started' and 'Completed'
* Ensure the icon is displayed correctly
--
* Start a course with a user in the frontend, finish a lesson
* Ensure the lesson status is completed with the correct icon

### Screenshots

<img width="285" alt="image" src="https://user-images.githubusercontent.com/176949/95096377-781c2d00-072c-11eb-999a-7f1a91dddd26.png">
 <img src="https://user-images.githubusercontent.com/176949/95109990-a81ffc00-073d-11eb-97ab-0f8602fd1ecf.png" />
